src/components/UnitAndLessonLists/UnitList/UnitListItem/UnitListItem.tsx

```diff
@@ -132,7 +145,7 @@ const UnitListItem: FC<UnitListItemProps> = (props) => {
         $gap={[8]}
         $pv={[8, 12]}
       >
-        {!isUnitOption && yearTitle && (
+        {!isUnitOption && yearTitle && !isExemplarUnit && (
```

Comment:
I'm confused as to why you are using a Paragraph tag which is then unstyled. This looks like it should be a heading tag ?

Reply:
I only added the '!isExemplarUnit' prop here - but yes can change to Heading makes sense.